### ISSUE TITLE
Examples

### ISSUE THREAD

**Issue by niteswim** [patriciogonzalezvivo/thebookofshaders#17]
Is it possible to put hints (examples) along with the exercises you give? Thanks!

**Comment by patriciogonzalezvivo** [comment]
True, I'm going to be teaching a class this semester using the book, from which I hope to improve the exercises and collect enough examples of them to make a gallery.
Thanks for noticing that.

**Comment by niteswim** [comment]
Sounds great, looking forward to it!
Thanks,
Madie

**Comment by redraw** [comment]
hey Patricio! this is a great book. I has started reading this book many times to refresh shader's concepts, but always got stucked on Chapter 05, until today that I've decided to understand it.

To improve that, I think you focus on the color sums, splitting them, suggesting to preview the result of each one i.e.
```
//color = (1.0-pct)*color;

//color = pct*vec3(0.0,1.0,0.0);

color = (1.0-pct)*color + pct*vec3(0.0,1.0,0.0);
```

The same with smoothstep() function, although you've already offered a help tip
```
// the original tip
float y = smoothstep(0.2,0.5,st.x) - smoothstep(0.5,0.8,st.x);

// making the values similar to the plot() function
float y = smoothstep(0.48,0.5,st.x) - smoothstep(0.5,0.52,st.x);
```

great work...

**Comment by yvan-sraka** [comment]
Hello! Because now we have a [gallery of examples](https://thebookofshaders.com/examples/), maybe we could close this problem? What do you think @niteswim?

**Comment by Delphinidae84** [comment]
I totally agree with @niteswim . As a complete beginner who is currently learning from this book I have to say that some kind of appendix with solutions (to the exercises) or at least some hints would be great! @yvan-sraka I'm not sure if the gallery is really enough here.
